doc/chapters/gateways/index.rst

```diff
@@ -302,12 +302,51 @@ An example configuration can look like this::
 Network gateway
 ===============
 
-The Network Gateway is used to provide Internet access through a container-specific firewall.
+The Network Gateway is used to setup network connection and configure which traffic is allowed and not.
 
 Configuration
 -------------
+The configuration is structured as a list of JSON objects that each describe rules for ``OUTGOING``
+or ``INCOMING`` network traffic. During evaluation the rules are read in order starting with the
```

Comment:
Is the order relevant given the "priority" stuff described below? If it's not relevant it would be nice to skip it as it is something the user hopefully should not assume and rely on.